### ISSUE TITLE
Inverted color not being shown

### ISSUE THREAD

**Issue by lucasvc** [jenkinsci/ansicolor-plugin#64]
As [read here](http://misc.flogisoft.com/bash/tip_colors_and_formatting), the escape sequence `\e[7m` allow to invert the colors. In example:

``` bash
RED='\e[31m'
INVERTED='\e[7m'
NORMAL='\e[0m'

if [ ! -f $HOME/_initilized ]; then
    echo -e "${RED}Some message\nMarking ${INVERTED}this text${NORMAL}."
    exit 1
fi
```

In bash `this text` is shown in red background and 'nocolor' foreground.
But the jenkins output is all in red color.

**Comment by Joe-Merten** [comment]
I think this could be closed due to #94. But maybe someone do some more testing before closing, e.g. even with different schemes etc.

**Comment by JoeMerten** [comment]
Fixed in 0.5.0
